lib/ngx/resp.md

```diff
@@ -87,6 +88,14 @@ status with an optional reason. The `reason` should be a string.
 
 [Back to TOC](#table-of-contents)
 
+bypass_if_checks
+------------------
+**syntax:** *ngx_resp.bypass_if_checks()*
+Bypasses the conditional header checks as defined in RFC 9110 for the current request.
+This method is useful when you need to skip validation of headers such as `If-Modified-Since`,
+`If-None-Match`, `If-Range`, `If-Unmodified-Since`, and `If-Match`, allowing the response to 
+proceed without evaluating these conditions.
```

Comment:
What about also writing down the `Why`? When people have to call this method?

Reply:
add desc